src/exodus/wallet.cpp

```diff
@@ -55,19 +55,19 @@ void Wallet::ClearSigmaMintChainState(
     e.groupId = 0;
     e.index = 0;
     e.block = -1;
-    e.isUsed = false;
+    e.tx = uint256();
```

Comment:
Same here.

Reply:
throw instead when try to set chain state to used coin.